www/config.yml

```diff
@@ -679,6 +679,14 @@ chapters:
             latitude: -5.7792569
             longitude: -35.200916
 
+    - name:      Newark, DE
+      email: newarkde@pyladies.com
+      meetup: PyLadies Newark DE
```

Comment:
Thanks for the PR! I think the `meetup` field should be the slug in the URL.
E.g. PyLadies Vancouver's meetup url is https://www.meetup.com/PyLadies-Vancouver/, so the slug is `PyLadies-Vancouver`.